### PR TITLE
Draft of USSRI blog entry.

### PR DESCRIPTION
**Pull request by jameshowison**

### CODE REVIEW

**Comment by danielskatz** [comment]
I have a bunch of edits/comments, but am not sure how best to make them into your fork.  A PR to you?

File: docs/ussri-blog/citeas-ussri-blog.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+![image: a citation.](an_article_citation.png)
+
+But if you have a piece of software, you very likely don't have the metadata needed to cite it. In part this is because with software there is no standard place to "write" the information, but it's also because authors sometimes want their users to cite something other than the piece of software directly. Examples include citing a paper that introduces the software (or demonstrated its potential), a published software manual or book, a "software paper" created specifically as a citation target, or a benchmarking paper.
+
+Great work is being done to guide best practices (including the [FORCE11 Working Group on Software Citation](https://www.force11.org/group/software-citation-working-group)) which recommends always including a direct citation to the software itself, including version numbers---something key for reproducibility---in addition to papers. We don't disagree, but we think it's important to let the authors decide how their contribution should be acknowledged and to link users with those requests.
```
Comment (katrinleinweber) [inline docs/ussri-blog/citeas-ussri-blog.md:16]: Should the `something` here be `sometimes`?

Comment (danielskatz) [reply docs/ussri-blog/citeas-ussri-blog.md:16]: no, something is right.  Sometimes would also be ok, but is weaker.

Comment (katrinleinweber) [reply docs/ussri-blog/citeas-ussri-blog.md:16]: Ah, thanks! If it's a synonymous expression for "a key aspect of", I get it now.

**Comment by jameshowison** [comment]
or could be better re-written as "something that is key for
reproducibility" :)

**Comment by danielskatz** [comment]
next step is to clean up the TBD section?

**Comment by jameshowison** [comment]
Yes, we need to expand some examples.

**Comment by danielskatz** [comment]
When do you think this might happen?  We would like to publish this next week, if it's available.

**Comment by jameshowison** [comment]
Working on it this afternoon, should have it to you next hour or so.

**Comment by jameshowison** [comment]
Ok, ready to go.  https://github.com/Impactstory/citeas-webapp/pull/13

**Pull request merged**